packages/js-web-sdk/test/umd.spec.ts

```diff
@@ -1,9 +1,9 @@
 import { OptimizelySDKWrapperConfig, OptimizelySDKWrapper } from '../src/OptimizelySDKWrapper'
 import testPublishedSDKWrapper from './testPublishedSdkWrapper'
 
-// The UMD script should have created jsWebSdk as a global variable
-declare namespace jsWebSdk {
+// The UMD script should have created optimizelySDK as a global variable
+declare namespace optimizelySDK {
```

Comment:
I think this is `optimizelySdk` in the `javascript-sdk`, probably should follow the same convention